.github/workflows/ci-pytorch-tests.yml

```diff
@@ -40,32 +40,36 @@ jobs:
+          - {cat: "quick", os: "ubuntu-20.04", pkg-name: "lightning", python-version: "3.10", pytorch-version: "1.12"}
+          - {cat: "quick", os: "windows-2022", pkg-name: "pytorch", python-version: "3.10", pytorch-version: "1.12"}
+          - {cat: "quick", os: "windows-2022", pkg-name: "lightning", python-version: "3.10", pytorch-version: "1.12"}
           # "oldest" versions tests, only on minimum Python
-          - {os: "macOS-11", pkg-name: "pytorch", python-version: "3.7", pytorch-version: "1.9", requires: "oldest"}
-          - {os: "macOS-11", pkg-name: "lightning", python-version: "3.7", pytorch-version: "1.9", requires: "oldest"}
-          - {os: "ubuntu-20.04", pkg-name: "pytorch", python-version: "3.7", pytorch-version: "1.9", requires: "oldest"}
-          - {os: "ubuntu-20.04", pkg-name: "lightning", python-version: "3.7", pytorch-version: "1.9", requires: "oldest"}
-          - {os: "windows-2022", pkg-name: "pytorch", python-version: "3.7", pytorch-version: "1.9", requires: "oldest"}
-          - {os: "windows-2022", pkg-name: "lightning", python-version: "3.7", pytorch-version: "1.9", requires: "oldest"}
+          - {cat: "quick", os: "macOS-11", pkg-name: "pytorch", python-version: "3.7", pytorch-version: "1.9", requires: "oldest"}
+          - {cat: "quick", os: "macOS-11", pkg-name: "lightning", python-version: "3.7", pytorch-version: "1.9", requires: "oldest"}
+          - {cat: "quick", os: "ubuntu-20.04", pkg-name: "pytorch", python-version: "3.7", pytorch-version: "1.9", requires: "oldest"}
+          - {cat: "quick", os: "ubuntu-20.04", pkg-name: "lightning", python-version: "3.7", pytorch-version: "1.9", requires: "oldest"}
+          - {cat: "quick", os: "windows-2022", pkg-name: "pytorch", python-version: "3.7", pytorch-version: "1.9", requires: "oldest"}
+          - {cat: "quick", os: "windows-2022", pkg-name: "lightning", python-version: "3.7", pytorch-version: "1.9", requires: "oldest"}
           # release-candidate tests, mixed Python versions
-          - {os: "macOS-11", pkg-name: "pytorch", python-version: "3.10", pytorch-version: "1.13", release: "pre"}
-          - {os: "ubuntu-20.04", pkg-name: "pytorch", python-version: "3.9", pytorch-version: "1.13", release: "pre"}
-          - {os: "windows-2022", pkg-name: "pytorch", python-version: "3.8", pytorch-version: "1.13", release: "pre"}
+          - {cat: "quick", os: "macOS-11", pkg-name: "pytorch", python-version: "3.10", pytorch-version: "1.13", release: "pre"}
+          - {cat: "quick", os: "ubuntu-20.04", pkg-name: "pytorch", python-version: "3.9", pytorch-version: "1.13", release: "pre"}
+          - {cat: "quick", os: "windows-2022", pkg-name: "pytorch", python-version: "3.8", pytorch-version: "1.13", release: "pre"}
+          # run test under SLOW label
+          - {cat: "long", os: "macOS-11", pkg-name: "pytorch", python-version: "3.7", pytorch-version: "1.11"}
```

Comment:
Why "long" when the variable name is "slow"?

Reply:
could be slow, but feel that slow is confusing as tests are not slow, just takes much longer...

Reply:
That's what slow means: 

"a: moving, flowing, or proceeding without speed or at less than usual speed"
"b: exhibiting or marked by low speed"

https://www.merriam-webster.com/dictionary/slow

Reply:
The fact, that it takes "much longer" (as @Borda said), doesn't necessarily mean, that it has low speed, but that the process has the same speed, but is long in the "distance" it needs to cover.

I'd go with `long`, mainly points c and d

3)
a : extending over a considerable time: _a long friendship_
b : having a specified duration: _two hours long_
c : prolonged beyond the usual time: _a long look_
d : lasting too long: _a long explanation_

https://www.merriam-webster.com/dictionary/long

Reply:
If you look at what's marked as "slow", they don't run more code (longer) but just spawn more processes (slower). Hence "slow"

In any case, the category for the job should match the job name and the workflow name. If you want to discuss renaming it to "long" then that should be done separately.

Reply:
Oh, sorry, my argument was more "academical" in the sense that `slow` and `long` are not really interchangeable. Anyway, I'm fine with both version (`long`/`slow`). But I fully agree, that if we go with `long`, then we need to change this in all the places at once (i.e. env variable, workflow name, etc.)

For that reason alone I'd say it makes more sense to go with `slow` as it is status quo and therefore less work

Reply:
I can make it `slow` if you want... :otter: